### PR TITLE
Update contribution information

### PR DESCRIPTION
**Pull request by TimFinucane**
Update contribution information, and some weirdness in scene (probably after merge) fixed, getting no fixable errors or warnings when scene is run.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<!--- Provide a general summary of the issue in the Title above -->
+
+### Expected Behavior
+<!--- Tell us what should happen -->
+
+### Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+### Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
```
Comment (cosmiccoincidence) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
<!--- Not obligatory, but suggest a causes or fixes for the bug -->
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+
+1.
+2.
+3.
+4.
```
Comment (joaoburatto) [inline .github/ISSUE_TEMPLATE/bug.md]: I don't feel this is needed.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+### Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+
+### Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+
+1.
+2.
+3.
```
Comment (joaoburatto) [inline .github/ISSUE_TEMPLATE/bug.md]: I don't think this is needed.

File: Assets/Scripts/Inventory/Inventory.cs [hunk 4]
```diff
@@ -82,7 +88,7 @@ public void CmdMoveItem(GameObject fromContainer, int fromIndex, GameObject toCo
         var to = toContainer.GetComponent<Container>();
 
         if (!Container.AreCompatible(to.GetSlot(toIndex), from.GetItem(fromIndex).itemType))
-            throw new System.Exception("Item not compatible with slot");
+            throw new InventoryOperationException("Item not compatible with slot");
```
Comment (Dyhr) [inline Assets/Scripts/Inventory/Inventory.cs]: I would rather that we don't use exceptions. They break the flow too much

Comment (TimFinucane) [reply Assets/Scripts/Inventory/Inventory.cs]: The use of exceptions in general should probably be a separate discussion.
I think this code does use exceptions in the way they are intended, as an exception only occurs if the code does not check whether or not the slot it's trying to move an item to is compatible (which is an exceptional circumstance).

File: .github/ISSUE_TEMPLATE/bug.md (hunk 3 above)
Comment (TimFinucane) [reply .github/ISSUE_TEMPLATE/bug.md]: It's there as an example to indicate the user should be listing the steps causing their bug

File: Assets/Scripts/Inventory/Inventory.cs (hunk 4 above)
Comment (Dyhr) [reply Assets/Scripts/Inventory/Inventory.cs]: Discussion is continued in Discord

**Pull request merged**